test/tcs_test/tcsmolten_salt_cases.h

```diff
@@ -22,4 +25,287 @@ int tcsmolten_salt_daggett(ssc_data_t &data)
 	return status;
 }
 
+// Power Tower molten salt with alternative Rankine Cycle configuration
+// Rankine Power Cycle: Sliding pressure & Air-cooled
+// Rest default configurations
+int tcsmolten_salt_daggett_sliding_pressure(ssc_data_t &data)
+{
+	tcsmolten_salt_default(data);
+
+	ssc_data_set_number(data, "tech_type", 3);
+	
+	int status = run_module(data, "tcsmolten_salt");
+
+	//single_owner_default(data);
+	//status += run_module(data, "singleowner");
+
```

Comment:
You don't need to replicate these two commented lines. Just keep in the first test and remove all the rest.